packages/web/components/complex/paginated-table.tsx

```diff
@@ -76,31 +76,34 @@ export const PaginatedTable = ({
           position: "relative",
         }}
       >
-        {virtualRows.map((virtualRow) => {
-          const row = rows[virtualRow.index] as Row<ObservablePoolWithMetric>;
-          return (
-            <Link
-              key={row.original.queryPool.id}
-              href={getPoolLink(row.original.queryPool)}
-              passHref
-              legacyBehavior
-            >
-              <a
-                style={{
-                  position: "absolute",
-                  top: 0,
-                  left: 0,
-                  width: "100%",
-                  height: `${virtualRow.size}px`,
-                  transform: `translateY(${virtualRow.start - topOffset}px)`,
-                }}
-              >
-                <MobileTableRow row={row} />
-              </a>
-            </Link>
-          );
-        })}
+{virtualRows.map((virtualRow) => {
+  const row = rows[virtualRow.index] as Row<ObservablePoolWithMetric>;
+
+  return row.original.queryPool.type === "transmuter" ? (
+    <Link
```

Comment:
Can't you just use a single `Link` component instead of two? You can use ternary operator inside `target` and `rel` props

Reply:
Link dont support target so i also cant use <a> thats why did this

Reply:
its working perfect still

Reply:
@HarjobandeepSingh I'm not sure I understand your comment, as you are using `target` as a prop here.

`<Link />` supports `target` as a prop in the latest version of NextJS https://nextjs.org/docs/app/api-reference/components/link#props

I agree with @DavideSegullo, it's easier to read if we just use one Link component and put the ternary in the `target` prop